### DISCUSSION TITLE
How to downgrade update

### DISCUSSION THREAD

**Discussion by Redteamer0101** [projectdiscovery/httpx#1616] [Q&A] [answered]
Hi Fellas,

I just updated httpx to latest version but now it is having issue in csv export it gives whole body.

How should I downgrade to previous version?

**Comment by olearycrew** [comment]
You can download any version you want here: https://github.com/projectdiscovery/httpx/releases

When you do if you could share at what version this problem starts, that would be great as I don't see anything in the release notes that indicates this should have changed.

**Comment by ehsandeep** [comment, marked as answer]
@Redteamer0101 the issue you experienced is now fixed in latest relese - https://github.com/projectdiscovery/httpx/releases/tag/v1.6.0
